go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/hashicorp/packer-plugin-vmware
 
 go 1.21.0
+toolchain go1.22.5
```

Comment:
I would suggest removing this toolchain argument tbh, I know that `go get` tends to add it automatically nowadays, I'm still not a fan of that, but that's probably because I don't understand the value in it if I'm honest.